pages/secret-manager/how-to/delete-secret.mdx

```diff
@@ -25,13 +27,14 @@ This page explains how to delete a [secret](/secret-manager/concepts/#secret) us
 
 1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
 2. Select the [region](/secret-manager/concepts/#region) in which to delete the secret, in the **Region** drop-down.
-3. Access the secret you wish to delete. The secret's **Overview** tab displays.
-4. Scroll down and click **Delete secret**. A pop-up displays.
-5. Type **DELETE** and click **Delete secret**.
+3. Click <Icon name="more" /> next to the secret you want to delete and click **Delete**. A pop-up displays informing you that the action schedules the deletion of your secret and its version.
+4. Type **DELETE** and click **Delete secret** to confirm. Your secret displays in the **Scheduled for deletion** tab for a period of 7 days before being permanently deleted.
 
 <Message type="important">
-  - Although you cannot delete a [protected secret](/secret-manager/concepts/#secret-protection), you can delete its versions
-  - Deleting a secret is a permanent action. It erases every version you have created for your secret
+  - Although you cannot delete a [protected secret](/secret-manager/concepts/#secret-protection), you can delete its versions.
+  - Deleting a secret is a permanent action. It erases every version you have created for your secret if you do not [recover it](/secret-manager/how-to/recover-secrets/) before the end of the retention period.
+  - Scheduled deletion does not delete your secret immediately unless it is an ephemeral secret. [Find out how to recover secrets scheduled for deletion](/secret-manager/how-to/recover-secrets/).
```

Comment:
@nerda-codes deletion is scheduled the same way for an ephemeral secret or a non-ephemeral one. We should remove line 36